rest_framework/request.py

```diff
@@ -341,7 +341,7 @@ def _parse(self):
             if media_type and is_form_media_type(media_type):
                 empty_data = QueryDict('', encoding=self._request._encoding)
             else:
-                empty_data = {}
+                empty_data = api_settings.DEFAULT_MISSING_DATA
```

Comment:
instead of exposing a new API can we just handle the empty data as None value instead of dict?

Reply:
That was my initial proposal, and I have no objection to doing so. However, please see the discussion in https://github.com/encode/django-rest-framework/pull/7195#issuecomment-589098125 (and the context that led up to this post).

Reply:
what I am suggesting you is based on this comment https://github.com/encode/django-rest-framework/pull/7199#issuecomment-1327335744 as Tom is reluctant to introduce new API surface to the framework. I saw those discussions as well and do agree with most of them.

Reply:
Yes, but Tom was *even more* reluctant to silently change the behavior ("too big of a behavior change" in [this](https://github.com/encode/django-rest-framework/pull/7195#issuecomment-588185418) earlier comment). He seemed to be more ok with adding the setting, see https://github.com/encode/django-rest-framework/pull/7195#issuecomment-588221965.

I have no stakes, and either way is fine for me. It's just think that this is question needs clarification between Tom and you, and I can't answer it.

Reply:
@tomchristie need your input here